### ISSUE TITLE
Support graphql-ruby 1.10.0

### ISSUE THREAD

**Issue by N0xFF** [xuorig/graphql-schema_comparator#28]
In https://github.com/rmosolgo/graphql-ruby/pull/2178 (graphql-ruby 1.10.0) was changed the `.from_definition` method which used in:
https://github.com/xuorig/graphql-schema_comparator/blob/722183a39f6d850ef58480fe78c56201bb060855/lib/graphql/schema_comparator.rb#L40-L41

So we can't compare even the same schemas:

```
> new_schema = BinaazSchema.to_definition
> GraphQL::SchemaComparator.compare(new_schema, new_schema)

Detected the following changes between schemas:

🛑  Schema mutation root has changed from `#<Class:0x00007f51e8424c08>` to `#<Class:0x00005616c7bfaf98>`
red
🛑  Schema query root has changed from `` to ``
red
```

**Comment by bnisbett** [comment]
Also experiencing this same behavioiur. Was about to open an issue also

**Comment by xuorig** [comment]
Just merged https://github.com/xuorig/graphql-schema_comparator/pull/29

I'll try `master` in our tooling and cut a release if I can validate it. Can y'all try `master` in the mean time? If not, I'll cut a release soon anyways :)

**Comment by xuorig** [comment]
Fixed a couple issues and tested against GitHub's schema. Looks like everything is 👍 with 1.10. 

I've released `1.0.0` https://github.com/xuorig/graphql-schema_comparator/releases/tag/v1.0.0

Let me know if you still have issues!

**Comment by N0xFF** [comment]
Tested in our project and it works. Thanks for quick fix.
